### ISSUE TITLE
Compounds similar to OSM-S-106 (Jan 2019 search)

### ISSUE THREAD

**Issue by mbhebhe** [OpenSourceMalaria/Series3#13]
![image](https://user-images.githubusercontent.com/26862286/51899903-17413b00-2408-11e9-8180-9ea9ba6797d2.png)

@cdsouthan did a search on compounds similar to our hit compound (OSM-S-106). There were 33 compounds; some are already in our OSM library, others are not commercially available and some are (picture above). So we need to decide which ones we should purchase and send to Dundee for biological evaluation.
See Issue #12 for more info

**Comment by cdsouthan** [comment]
Could @holeung manage a 3D overlay?

**Comment by MFernflower** [comment]
My vote is for these two molecules based on cost and overall lookalikeness of them to the s3 hit 
![_vote](https://user-images.githubusercontent.com/3164942/51952223-a8fb7900-2405-11e9-95de-c075db90708d.png)

**Comment by mbhebhe** [comment]
I agree with @MFernflower. Although in the past not having a primary sulfonamide has reduced potency. I think doing a 3D overlay would be good.

**Comment by MFernflower** [comment]
I feel it's important to note that the only one with an exposed sulfonamide is going to cost upwards of 2000 usd to screen against malaria (unless @mbhebhe can take a shot at making it?)

**Comment by MFernflower** [comment]
I'd like to hear from @mbhebhe and/or @mattodd regarding a final opinion on this - I still think sending a few select or all? of the Enamine LTD compounds listed would be the best option

**Comment by MFernflower** [comment]
@holeung Would it be possible to dock some of these drugs to your model of the TRNA synthase?

**Comment by mbhebhe** [comment]
Sending all the enamine compounds is a good idea as they are not expensive.

**Comment by cdsouthan** [comment]
Fine, Ukraine chemists are good value :)  However, best to do a precautionary independent purity check (but only later on the good hits)

**Comment by MedChemProf** [comment]
I second @cdsouthan 's purity check with one additional caveat. Multiple times I have seen hits from commercial vendors repeat and have a good purity profile via LCMS, but then subsequently fail following testing after in-house synthesis. Not a deal breaker, just something to be cognizant about.

**Comment by cdsouthan** [comment]
Ho hum - so pure but wrong?   (welcome back by the way :)

**Comment by drc007** [comment]
As they say "Trust but verify".

**Comment by MFernflower** [comment]
Always good to verify your hits - especially when the hit is from a commercial library!

**Comment by mattodd** [comment]
Ok, so
1) There are two compounds we're interested in? What's the cost? Need 5 or 10 mg, roughly.
2) We ship to UCL, NMR, re-ship 2 mg to Dundee?
M

On Wed., 20 Mar. 2019, 19:41 MFernflower, <notifications@github.com<mailto:notifications@github.com>> wrote:

Always good to verify your hits - especially when the hit is from a commercial library!

**Comment by MFernflower** [comment]
Price for the two molecules I proposed is gonna be 178 usd before shipping
(but i think enamine has a UK distributor)

https://www.enaminestore.com/catalog/Z196159998

https://www.enaminestore.com/catalog/Z373297698


I tagged this needs action because it seemed like there was a resurgence in
intrest regarding this topic
